src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
```

Comment:
Consider adding a class to the notification element for easier styling and manipulation via CSS. This can help in maintaining consistent styles across different notifications.